src/main/java/com/googleinterns/zoomtube/data/Lecture.java

```diff
@@ -23,14 +23,6 @@
 @GenerateTypeAdapter
 @AutoValue
 public abstract class Lecture {
-  /** Used to create a Lecture entity and its fields. */
-  // TODO: Move to LectureEntityCreator class.
-  public static final String ENTITY_KIND = "Lecture";
-  public static final String PROP_NAME = "lectureName";
-  public static final String PROP_VIDEO_URL = "videoUrl";
-  public static final String PROP_VIDEO_ID = "video-id";
```

Comment:
I thought this code was not merged yet? Or was it?

Reply:
That is old code that I am now removing